docs/modules/integrations/partials/authentication.adoc

```diff
@@ -0,0 +1,39 @@
+== Authentication
+
+* if authentication through basic schema is available on your jira instance use `jira.username` and `jira.password` properties, note that in this case both properties must be set
+* in case of two-factor authentication or any other complex authentication process it's possible to set session cookies into request headers like the following example shows
```

Comment:
```suggestion
* in case of two-factor authentication or any other complex authentication process it's possible to set session cookies into request headers like the following example shows (`jira.username` and `jira.password` properties must be empty in this case)
```